hs_file_types/utils.py

```diff
@@ -226,20 +231,159 @@ def get_logical_file_type(res, file_id, hs_file_type=None, fail_feedback=True):
     return logical_file_type_class
 
 
+def get_logical_file(agg_type_name):
+    file_type_map = {"GeographicRasterAggregation": GeoRasterLogicalFile,
+                     "SingleFileAggregation": GenericLogicalFile,
+                     "FileSetAggregation": FileSetLogicalFile,
+                     "MultidimensionalAggregation": NetCDFLogicalFile,
+                     "GeographicFeatureAggregation": GeoFeatureLogicalFile,
+                     "ReferencedTimeSeriesAggregation": RefTimeseriesLogicalFile,
+                     "TimeSeriesAggregation": TimeSeriesLogicalFile,
+                     }
+    return file_type_map[agg_type_name]
+
+
+def is_aggregation_metadata_file(file):
+    return file.name.endswith('_meta.xml')
+
+
+def is_map_file(file):
+    return file.name.endswith('_resmap.xml') or file.name == 'resourcemap.xml'
+
+
+def is_resource_metadata_file(file):
+    return os.path.basename(file.name) == 'resourcemetadata.xml'
+
+
+def identify_and_ingest_metadata_files(resource, files):
+    from hs_file_types.utils import identify_metadata_files
+    res_files, meta_files = identify_metadata_files(files)
```

Comment:
Double check the imports here. It looks to me these imports are not needed since methods are in the same file.

Reply:
lol, a side effect of refactoring.  Removing the imports.